elide-core/src/main/java/com/yahoo/elide/core/TransactionRegistry.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.core;
+import java.util.Set;
+/**
+ * Database interface library.
+ */
+public interface TransactionRegistry {
+
+    public static class TransactionEntry {
```

Comment:
I think we might get build warnings for the lack of javadoc for the inner static class - and also the public methods below.  Can we add a bit of documentation?